### ISSUE TITLE
[5.x]: Sources sidebar not accessible via keyboard [A11y]

### ISSUE THREAD

**Issue by jakobhaerter** [craftcms/cms#15876]
### Description

The sources sidebar is focusable but pressing `space` or `enter` does not change the view. A JS error is being thrown.

![Image](https://github.com/user-attachments/assets/4c3da6a6-0b72-4eb5-966b-2a29b21452d4)

### Steps to reproduce

1. Go to the entries view
2. Navigate to the sources sidebar via the keyboard
3. Try to open a tab with the `space` or `enter` key

### Expected behavior

Pressing `space` or `enter` should update the view.

### Actual behavior

No view update but a JS error inside the console.

### Craft CMS version

5.4.5.1

### Installed plugins and versions

-

**Comment by i-just** [comment]
Hi, thanks for reporting! I raised a PR for this.

**Comment by brandonkelly** [comment]
Craft 5.4.8 is out with that fix. Thanks again!

**Comment by jakobhaerter** [comment]
Thank you, @i-just & @brandonkelly! ✨
